tests/mutation_test.cpp

```diff
@@ -0,0 +1,116 @@
+#include "catch/catch.hpp"
+
+#include "mutation.h"
+
+#include "game.h"
+#include "player.h"
+
+void clear_mutations( player &p )
```

Comment:
If you used a temporary NPC instead of a complete player, this wouldn't be needed.